### ISSUE TITLE
Value of type 'Moment' has no member 'substract'

### ISSUE THREAD

**Issue by AndyDunn** [n8armstrong/CalendarView#5]
I'm getting the following error when trying to compile a project including CalendarView:

```
Value of type 'Moment' has no member 'substract'
/Pods/CalendarView/CalendarView/CalendarView/Classes/ContentView.swift:43:23: 
```

It happens on this line of ContentView.swift:

```
var currentDate = date.substract(1, .Months)
```

It's using Moment version 0.3 and CalendarView 1.1.0.

Any ideas?

**Comment by armstrongnate** [comment]
Yes. It uses [SwiftMoment](https://github.com/akosma/SwiftMoment) and that framework has been updated. One of the fixes is that it changed `substract` to `subtract`. I'll try to get to it this weekend.

**Comment by AndyDunn** [comment]
Right, that would make sense. Thanks for letting me know. I'll maybe fix that locally until you have it in place in the library.

**Comment by castaldichristian** [comment]
Following up with this issues, I am also getting a few errors when I include the library into the project, beside the error mentioned above I am getting

 `/Pods/CalendarView/CalendarView/CalendarView/Classes/MonthView.swift:90:38: Type of expression is ambiguous without more context`

Thanks

**Comment by armstrongnate** [comment]
Thank you @AndyDunn and @kriskast. I just released `1.1.1` which should fix those errors.
